SparkPost/2web2ui
# EO-871: Updated list results styling

### What Changed
 - Updated the styles of list results

### How To Test
 - Go to `/recipient-validation/list`
 - Upload a list
 - Check that the card on the bottom now matches the styles from the mockup

### Notes
 - File Name omitted for now until API update
 - Date moved to left
 - Button kept on right
 - Button text changed to `Download Results`
 - Title changed to `Last Validation`

### To Do
